package.json

```diff
@@ -62,6 +62,7 @@
   "peerDependencies": {
     "react": "~0.13.x || ~0.14.x || ^15.0.0"
   },
+  "types": "./react-day-picker.d.ts",
```

Comment:
i'd say this belongs up near `main` and `style` keys as it's a distributable file